pkgs/tools/security/iaito/default.nix

```diff
@@ -1,61 +1,65 @@
 { lib
 , stdenv
 , fetchFromGitHub
+, meson
+, ninja
 , pkg-config
+, python3
 , qtbase
 , qttools
 , radare2
 , wrapQtAppsHook
-, zip
-, nix-update-script
 }:
 
+# TODO MacOS support.
+# TODO Build and install translations.
+
 stdenv.mkDerivation rec {
   pname = "iaito";
-  version = "5.7.0";
+  version = "5.7.2";
 
   src = fetchFromGitHub {
     owner = "radareorg";
     repo = pname;
     rev = version;
-    fetchSubmodules = true;
-    sha256 = "sha256-qEJTsS669eEwo2iiuybN72O5oopCaGEkju8+ekjw2zk=";
+    sha256 = "sha256-5/G5wfdc6aua90XLP3B7Ruy8F3NTXzWfQE6yVDZ0rX8=";
   };
 
-  nativeBuildInputs = [ pkg-config qttools wrapQtAppsHook zip ];
+  nativeBuildInputs = [ meson ninja pkg-config python3 qttools wrapQtAppsHook ];
+
   buildInputs = [ radare2 qtbase ];
 
-  postPatch = ''
-    substituteInPlace Makefile \
-      --replace "git submodule update --init" ""
+  postUnpack = ''
+    sourceRoot=$sourceRoot/src
   '';
```

Comment:
This can be changed to `sourceRoot = "src"`.

Reply:
I've tried that, It didn't work. I've asked people around on IRC and they suggested me to use this.